src/main/java/org/isf/priceslist/manager/PriceListManager.java

```diff
@@ -42,97 +42,100 @@
 
 	@Autowired
 	private PricesListIoOperations ioOperations;
-	
+
 	/**
-	 * Return the list of {@link PriceList}s in the DB
-	 * @return the list of {@link PriceList}s
-	 * @throws OHServiceException 
+	 * Return the list of {@link List}s in the DB
+	 *
+	 * @return the list of {@link List}s
+	 * @throws OHServiceException
 	 */
 	public ArrayList<PriceList> getLists() throws OHServiceException {
-        return ioOperations.getLists();
+		return ioOperations.getLists();
 	}
-	
+
 	/**
 	 * Return the list of {@link Price}s in the DB
+	 *
 	 * @return the list of {@link Price}s
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
 	public ArrayList<Price> getPrices() throws OHServiceException {
-        return ioOperations.getPrices();
+		return ioOperations.getPrices();
 	}
 
 	/**
-	 * Updates all {@link Price}s in the specified {@link PriceList}
-	 * @param list - the {@link PriceList}
+	 * Updates all {@link Price}s in the specified {@link List}
+	 *
+	 * @param list - the {@link List}
 	 * @param prices - the list of {@link Price}s
 	 * @return <code>true</code> if the list has been replaced, <code>false</code> otherwise
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
 	public boolean updatePrices(PriceList list, ArrayList<Price> prices) throws OHServiceException {
-        return ioOperations.updatePrices(list, prices);
+		return ioOperations.updatePrices(list, prices);
 	}
 
 	/**
-	 * Insert a new {@link PriceList} in the DB
-	 * 
-	 * @param list - the {@link PriceList}
+	 * Insert a new {@link List} in the DB
+	 *
+	 * @param list - the {@link List}
 	 * @return <code>true</code> if the list has been inserted, <code>false</code> otherwise
-	 * @throws OHServiceException 
+	 * @throws OHServiceException
 	 */
 	public boolean newList(PriceList list) throws OHServiceException {
-	    validatePriceList(list);
-        return ioOperations.newList(list);
+		validatePriceList(list);
+		return ioOperations.newList(list);
 	}
 
 	/**
-	 * Update a {@link PriceList} in the DB
-	 * 
-	 * @param updateList - the {@link PriceList} to update
+	 * Update a {@link List} in the DB
+	 *
+	 * @param updateList - the {@link List} to update
```

Comment:
like above

Reply:
I believe I've resolved (merged) the changes from OP-399